archlinuxcn/discord/lilac.yaml

```diff
@@ -4,11 +4,14 @@ maintainers:
   - github: SilverRainZ
     email: Shengyu Zhang <la@archlinuxcn.org>
 
-build_prefix: archlinuxcn-x86_64
+build_prefix: extra-x86_64
 
 pre_build: aur_pre_build
 
 post_build: aur_post_build
 
 update_on:
   - aur: discord
+
+depends:
```

Comment:
FYI: This key is renamed to `repo_depends` a few days ago.

https://github.com/archlinuxcn/repo/pull/1090

Reply:
Sad to hear that :cry: .

Reply:
Fixed in 7ee39a401.